### ISSUE TITLE
Use CIRCLE_TOKEN from a bot

### ISSUE THREAD

**Issue by isuruf** [conda-forge/staged-recipes#4780]
Currently the token of @pelson is used, but it's better to use a token from a bot.

**Comment by jakirkham** [comment]
Certainly open to it as long as Phil's ok with it. Any bot you would recommend?

**Comment by pelson** [comment]
Yep, definitely. Do we have documentation for which bot (/user) does what? If not, may I ask that we do that in order to get a good picture of the most appropriate bot in this instance?
With that in place, I'd be more than happy to set up the token.

**Comment by isuruf** [comment]
Here are the things that I know,

- Linting Github - conda-forge-linter
- Register Circle - pelson
- Register Appveyor - conda-forge-admin (Some repos are registered with conda-forge-coordinator) - appveyor uses the token of the user that created the appveyor hook for future GH operations
- Register Github - conda-forge-manager

**Comment by pelson** [comment]
conda-forge-manager looks like a good choice then.
Are you happy to put these contents forwards as a documentation PR, and I'll happily review, merge, and setup the token. 👍
